enterprise/server/scim/scim_test.go

```diff
@@ -511,6 +521,16 @@ func TestDeleteUser(t *testing.T) {
 		require.True(t, status.IsNotFoundError(err))
 	}
 
+	// Delete user US104 using a DELETE request.
+	{
+		code, body := tc.Delete(baseURL + "/scim/Users/US104")
+		require.Equal(tc.t, http.StatusNoContent, code, "body: %s", string(body))
+		require.NoError(t, err)
+		_, err = udb.GetUserByID(userCtx, "US104")
+		require.Error(t, err)
```

Comment:
nit: for the sake of illustration, maybe also assert that the user exists before doing `tc.Delete` ?